File: src/sdk-reference/java/1/document/validate/snippets/validate.java
try {
  boolean valid = kuzzle.getDocument().validate("nyc-open-data", "yellow-taxi", "{" +
    "\"capacity\": 4" +
  "}");

  if (valid) {
    System.out.println("The document is valid");
  }
} catch (KuzzleException e) {
  System.err.println(e.getMessage());
}

File: src/sdk-reference/csharp/1/document/validate/snippets/validate.cs
try {
  bool valid = kuzzle.document.validate("nyc-open-data", "yellow-taxi", @"{
    ""capacity"": 4
  }");

  if (valid) {
    Console.WriteLine("The document is valid");
  }
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.getMessage());
}
